plaso/parsers/plist_plugins/launchd.py

```diff
@@ -56,12 +56,7 @@ class MacOSLaunchdPlistPlugin(interface.PlistPlugin):
   # /Library/LaunchAgents/*.plist
   # ~/Library/LaunchAgents
 
-  PLIST_KEYS = frozenset([
-      'GroupName',
-      'Label',
-      'Program',
-      'ProgramArguments',
-      'UserName'])
+  PLIST_KEYS = frozenset(['Label'])
```

Comment:
@Spferical This will cause this plugin to match any plist that has a `Label` key in the root item is that really the desired behavior?

Reply:
It might lead to false positives and the default plugin not to be applied if no suitable plugin is found. I've made some changes. PTAL